source/tests/system/nirfmxnr_driver_api_tests.cpp

```diff
@@ -722,6 +715,98 @@ TEST_F(NiRFmxNRDriverApiTests, TxpContiguousMultiCarrierFromExample_FetchData_Da
   EXPECT_NE(0.0, txp_fetch_power_trace_response.power(0));
 }
 
+// Note: there aren't any complex attributes in attributes.py, but this at least exercises the code.
```

Comment:
Note: The actual changes start here. Most of the above changes (except new includes) were auto-formatting changes.